ERROR 2015-08-17 15:04:44,200 [12] ADOExceptionReporter   - Table definition has changed, please retry transaction
ERROR 2015-08-17 15:04:44,221 [12] ADOExceptionReporter   - Table definition has changed, please retry transaction
ERROR 2015-08-17 15:04:44,229 [12] BaseController         - Error: [/Scheduler/EmailTodos?currentTime=20]<<could not load an entity: [RadialReview.Models.UserOrganizationModel#74][SQL: SELECT userorgani0_.ResponsibilityGroupModel_id as Id18_0_, userorgani0_1_.DeleteTime as DeleteTime18_0_, userorgani0_1_.Organization_id as Organiza3_18_0_, userorgani0_.IsRadialAdmin as IsRadial2_22_0_, userorgani0_.CountPerPage as CountPer3_22_0_, userorgani0_.ManagingOrganization as Managing4_22_0_, userorgani0_.ManagerAtOrganization as ManagerA5_22_0_, userorgani0_.AttachTime as AttachTime22_0_, userorgani0_.CreateTime as CreateTime22_0_, userorgani0_.DetachTime as DetachTime22_0_, userorgani0_.DeleteTime as DeleteTime22_0_, userorgani0_.EmailAtOrganization as EmailAt10_22_0_, userorgani0_.JobDescription as JobDesc11_22_0_, userorgani0_.JobDescriptionFromTemplateId as JobDesc12_22_0_, userorgani0_.TempUser_id as TempUser13_22_0_, userorgani0_.Cache_id as Cache14_22_0_, userorgani0_.User_id as User15_22_0_ FROM `UserOrganizationModel` userorgani0_ inner join `ResponsibilityGroupModel` userorgani0_1_ on userorgani0_.ResponsibilityGroupModel_id=userorgani0_1_.Id WHERE userorgani0_.ResponsibilityGroupModel_id=?]>>
NHibernate.Exceptions.GenericADOException: could not load an entity: [RadialReview.Models.UserOrganizationModel#74][SQL: SELECT userorgani0_.ResponsibilityGroupModel_id as Id18_0_, userorgani0_1_.DeleteTime as DeleteTime18_0_, userorgani0_1_.Organization_id as Organiza3_18_0_, userorgani0_.IsRadialAdmin as IsRadial2_22_0_, userorgani0_.CountPerPage as CountPer3_22_0_, userorgani0_.ManagingOrganization as Managing4_22_0_, userorgani0_.ManagerAtOrganization as ManagerA5_22_0_, userorgani0_.AttachTime as AttachTime22_0_, userorgani0_.CreateTime as CreateTime22_0_, userorgani0_.DetachTime as DetachTime22_0_, userorgani0_.DeleteTime as DeleteTime22_0_, userorgani0_.EmailAtOrganization as EmailAt10_22_0_, userorgani0_.JobDescription as JobDesc11_22_0_, userorgani0_.JobDescriptionFromTemplateId as JobDesc12_22_0_, userorgani0_.TempUser_id as TempUser13_22_0_, userorgani0_.Cache_id as Cache14_22_0_, userorgani0_.User_id as User15_22_0_ FROM `UserOrganizationModel` userorgani0_ inner join `ResponsibilityGroupModel` userorgani0_1_ on userorgani0_.ResponsibilityGroupModel_id=userorgani0_1_.Id WHERE userorgani0_.ResponsibilityGroupModel_id=?] ---> MySql.Data.MySqlClient.MySqlException: Table definition has changed, please retry transaction
   at MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   at MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   at MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   at MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   at MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   at NHibernate.AdoNet.AbstractBatcher.ExecuteReader(IDbCommand cmd)
   at NHibernate.Loader.Loader.GetResultSet(IDbCommand st, Boolean autoDiscoverTypes, Boolean callable, RowSelection selection, ISessionImplementor session)
   at NHibernate.Loader.Loader.DoQuery(ISessionImplementor session, QueryParameters queryParameters, Boolean returnProxies)
   at NHibernate.Loader.Loader.DoQueryAndInitializeNonLazyCollections(ISessionImplementor session, QueryParameters queryParameters, Boolean returnProxies)
   at NHibernate.Loader.Loader.LoadEntity(ISessionImplementor session, Object id, IType identifierType, Object optionalObject, String optionalEntityName, Object optionalIdentifier, IEntityPersister persister)
   --- End of inner exception stack trace ---
   at NHibernate.Loader.Loader.LoadEntity(ISessionImplementor session, Object id, IType identifierType, Object optionalObject, String optionalEntityName, Object optionalIdentifier, IEntityPersister persister)
   at NHibernate.Loader.Entity.AbstractEntityLoader.Load(ISessionImplementor session, Object id, Object optionalObject, Object optionalId)
   at NHibernate.Loader.Entity.AbstractEntityLoader.Load(Object id, Object optionalObject, ISessionImplementor session)
   at NHibernate.Persister.Entity.AbstractEntityPersister.Load(Object id, Object optionalObject, LockMode lockMode, ISessionImplementor session)
   at NHibernate.Event.Default.DefaultLoadEventListener.LoadFromDatasource(LoadEvent event, IEntityPersister persister, EntityKey keyToLoad, LoadType options)
   at NHibernate.Event.Default.DefaultLoadEventListener.DoLoad(LoadEvent event, IEntityPersister persister, EntityKey keyToLoad, LoadType options)
   at NHibernate.Event.Default.DefaultLoadEventListener.Load(LoadEvent event, IEntityPersister persister, EntityKey keyToLoad, LoadType options)
   at NHibernate.Event.Default.DefaultLoadEventListener.OnLoad(LoadEvent event, LoadType loadType)
   at NHibernate.Impl.SessionImpl.FireLoad(LoadEvent event, LoadType loadType)
   at NHibernate.Impl.SessionImpl.ImmediateLoad(String entityName, Object id)
   at NHibernate.Proxy.AbstractLazyInitializer.Initialize()
   at NHibernate.Proxy.AbstractLazyInitializer.GetImplementation()
   at NHibernate.Proxy.DefaultLazyInitializer.Intercept(InvocationInfo info)
   at UserOrganizationModelProxy.get_User()
   at RadialReview.Controllers.SchedulerController.<EmailTodos>b__1(UserOrganizationModel y) in c:\repos\Radial\RadialReview\RadialReview\Controllers\SchedulerController.cs:line 59
   at RadialReview.ObjectExtensions.NotNull[T,R](T obj, Func`2 f) in c:\repos\Radial\RadialReview\RadialReview\Utilities\Extensions\ObjectExtensions.cs:line 30
   at RadialReview.Controllers.SchedulerController.<EmailTodos>b__0(TodoModel x) in c:\repos\Radial\RadialReview\RadialReview\Controllers\SchedulerController.cs:line 59
   at System.Linq.Lookup`2.Create[TSource](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.GroupedEnumerable`3.GetEnumerator()
   at RadialReview.Controllers.SchedulerController.<EmailTodos>d__1c.MoveNext() in c:\repos\Radial\RadialReview\RadialReview\Controllers\SchedulerController.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at lambda_method(Closure , Task )
   at System.Threading.Tasks.TaskHelpersExtensions.ThrowIfFaulted(Task task)
   at System.Web.Mvc.Async.TaskAsyncActionDescriptor.EndExecute(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass34.<BeginInvokeAsynchronousActionMethod>b__33(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.Async.AsyncResultWrapper.End[TResult](IAsyncResult asyncResult, Object tag)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3c()
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass45.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3e()
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass30.<BeginInvokeActionMethodWithFilters>b__2f(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.Async.AsyncResultWrapper.End[TResult](IAsyncResult asyncResult, Object tag)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass1e.<>c__DisplayClass28.<BeginInvokeAction>b__19()
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass1e.<BeginInvokeAction>b__1b(IAsyncResult asyncResult)
ERROR 2015-08-17 15:04:44,268 [45] BaseAccessor           - Scheduled task error. 322
System.Net.WebException: The remote server returned an error: (500) Internal Server Error.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   at System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RadialReview.Accessors.TaskAccessor.<ExecuteTask>d__4.MoveNext() in c:\repos\Radial\RadialReview\RadialReview\Accessors\TaskAccessor.cs:line 116
INFO  2015-08-17 16:01:31,338 [46] BaseController         - Organization: [/Measurable/BlankEditorRow?_=1439844997180] --> [/Measurable/BlankEditorRow?_=1439844997180]
INFO  2015-08-17 16:37:52,657 [20] BaseController         - Organization: [/Manage/UserDetails/948] --> [/Manage/UserDetails/948]
INFO  2015-08-17 17:07:20,365 [11] BaseController         - Organization: [/Manage/UserDetails/948] --> [/Manage/UserDetails/948]
